### ISSUE TITLE
Testing Pretrained Imagenet Models

### ISSUE THREAD

**Issue by purple-panda** [rbgirshick/py-faster-rcnn#242]
Hello!

This is my first time experimenting with caffe so I apologize in advance if this is very simple.

I was able to successfully run the demo. I was also able to successfully run 
`./tools/test_net.py --def models/pascal_voc/VGG16/faster_rcnn_alt_opt/faster_rcnn_test.pt --net data/faster_rcnn_models/VGG16_faster_rcnn_final.caffemodel --vis`

Now I'm trying to test the pre-trained imagenet models provided. I've tried the following commands
`./tools/test_net.py --def models/pascal_voc/VGG16/faster_rcnn_alt_opt/faster_rcnn_test.pt --net data/imagenet_models/VGG16.v2.caffemodel --vis`

`./tools/test_net.py --def models/pascal_voc/VGG16/faster_rcnn_end2end/test.prototxt --net data/imagenet_models/VGG16.v2.caffemodel`

The script runs without error but it looks like the VGG model isn't actually detecting anything. Should I be using a different prototext file? What would that be?

**Comment by IdiosyncraticDragon** [comment]
The VGG16.v2.caffemodel trained for classification, it surely can not detect any things.

**Comment by ednarb29** [comment]
Right, the model is just for initialization, that you don't need to train from scratch. There are some papers out there that describe learning on pre-trained models due to their generalisation in the early layers.

**Comment by ashutalewar** [comment]
@purple-panda - were you able to run the demo using imagenet pretrained VGG model? I am also trying the same thing and facing a similar issue.
